basic-sql-operations.md

```diff
@@ -1,174 +1,156 @@
 ---
-title: Explore SQL with TiDB
+title: Basic SQL operations
```

Comment:
Please keep the title consistent with the 1st-level heading in the body.
"operations" > "Operations"